docs/en/observability/create-alerts.asciidoc

```diff
@@ -3,39 +3,72 @@
 
 [IMPORTANT]
 ====
-Ensure Alerting is already set up in {kib}. For details, see
+Make sure alerting is already set up in {kib}. For details, see
 {kibana-ref}/alerting-setup.html[Setup and prerequisites].
 ====
 
-Alerting enables you to detect complex *conditions* defined by a rule
+Alerting enables you to detect complex conditions defined by a *rule*
 within the Logs, Metrics, Uptime, and APM apps. When a condition is met, the rule
 tracks it as an *alert* and responds by triggering one or more *actions*.
 
-Listed on the <<view-observability-alerts,Alerts>> page are all of your
-observability alerts that have met a condition defined by a rule you created
-using the Logs, Metrics, Uptime, or APM apps.
+Alerts and rules related to {observability} apps including Logs, Metrics, Uptime, and APM can be managed in the {observability} UI.
+You can also manage {observability} app rules alongside rules for other apps from the {kibana-ref}/create-and-manage-rules.html[{kib} Management UI].
 
-IMPORTANT: Not all of the predefined rules in Stack Management will generate and 
-list an alert on the Alerts page. Only alerts generated by rules relating to Logs, Metrics, Uptime, and APM
-can be viewed on the Alerts page.
+[discrete]
+[[create-alerts-rules]]
+== Create rules
+
+The first step when setting up alerts is to create a rule.
+To create and manage rules related to {observability} apps,
+go to the {observability} *Alerts* page and click *Manage Rules* to navigate to the {observability} Rules page.
+
+You can also create rules directly from the Logs, Metrics, Uptime, and APM apps without leaving the app by
+clicking *Alerts and rules* and selecting a rule, or you can select *Manage Rules* to go to the {observability} Alerts page.
```

Comment:
But clicking *Manage Rules* here takes you to the **Alerts page**? That seems goofy.

Reply:
Clicking *Manage Rules* doesn't take you to the **Alerts page**, but to the **Rules** page under Alerts page if that makes sense. 

In the Video you can see that both `Manage rules` link in the *Alerts page* and `Manage rules` link in the top header popup of the o11y apps takes you to the **new o11y Rules page** (and not to the old Stack Management > Rules and connectors page)

https://user-images.githubusercontent.com/2852703/162902202-8748432d-c08f-4cc3-b128-d04ff998455d.mov

Reply:
Aha! This makes a lot more sense. Thanks, @mgiota.

Reply:
🙈  thanks @mgiota @bmorelli25!